### ISSUE TITLE
rvm install error, can someone please. help me 

### ISSUE THREAD

**Issue by newlearnerfor** [rvm/rvm#5041]
==> Homebrew has enabled anonymous aggregate formulae and cask analytics.
Read the analytics documentation (and how to opt-out) here:
  https://docs.brew.sh/Analytics
No analytics data has been sent yet (or will be during this `install` run).

==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations

==> Next steps:
- Add Homebrew to your PATH in /Users/learner/.zprofile:
    echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/learner/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
- Run `brew help` to get started
- Further documentation: 
    https://docs.brew.sh
**Something went wrong during Homebrew installation,
can not find 'brew' command, please report a bug: https://github.com/rvm/rvm/issues
Requirements installation failed with status: 1.**

**Comment by newlearnerfor** [comment]
MacOS big sur version 11.1

**Comment by newlearnerfor** [comment]
can someone please help me

**Comment by newlearnerfor** [comment]
@pedrofurtado  can you please help on this ^^^

**Comment by pedrofurtado** [comment]
Hey, @newlearnerfor ! 👋 

I really did not understand the error and your situation. 

Can you explain for us the scenario? Are you trying to install ruby, install rvm itself, upgrade rvm version?

What commands did you executed in terminal, and what was the output of each one?

Let me know the details and we will be able to help you with this 🤝

**Comment by pedrofurtado** [comment]
What is the version of your rvm? You can check it with the command `rvm -v` 👍

**Comment by newlearnerfor** [comment]
its rvm 1.29.12 @pedrofurtado 
Thank you

**Comment by pedrofurtado** [comment]
This is strange 🤔 

Can you follow these steps to confirm if error persists?

1. Execute `rvm implode`, to remove rvm installation
2. Close the terminal and open it again
3. If you do not have brew installed, please install it with the instructions from https://brew.sh/
4. Install again rvm, follow these steps:
```bash
command curl -ksSL https://rvm.io/mpapis.asc | gpg2 --import -
command curl -ksSL https://rvm.io/pkuczynski.asc | gpg2 --import -
curl -ksSL https://get.rvm.io | bash -s stable
source /etc/profile.d/rvm.sh
```
5. Then, execute `rvm install 2.6.3`

let me know your results 🤝

**Comment by newlearnerfor** [comment]
@pedrofurtado  till step 4 it's good and when i try to install step 5 
I got an error, tried looking for some answers but couldn't figure out 
**Error**
Error running '__rvm_make -j8',
please read /Users/learner.rvm/log/1611946463_ruby-2.6.3/make.log

There has been an error while running make. Halting the installation.

**Comment by newlearnerfor** [comment]
@pedrofurtado but when I check the ruby version its 
 **test_app % ruby -v**
ruby 2.6.3p62

**Comment by pedrofurtado** [comment]
Oh, now I understand. Thanks for feedback @newlearnerfor!

Well, we of rvm team are debugging the root cause of this problem, specifically in MacOS Big Sur.

Unfortunately, your problem is the same of these other issues: https://github.com/rvm/rvm/issues?q=is%3Aissue+is%3Aopen+label%3A%22MacOS+Big+Sur+bug%22

For a while, there is not a final official solution. But there is some workarounds, provided by community, on the issues above 🤝 

If you have any workaround, or if you can provide us a PR with this fix, we will be glad in review it 🎉 🤝

**Comment by newlearnerfor** [comment]
@pedrofurtado 
Thank you :) 🙏

**Comment by Mrzhaohan** [comment]
I also have the same problem. I have been working for 2 days now, and the previous steps are all correct. Finally, I failed in the error report of RVM install 2.2.10.

The core error：
----------------------------------------------------------------------------------------------
closure.c:264:14: error: implicit declaration of function 'ffi_prep_closure' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    result = ffi_prep_closure(pcl, cif, callback, (void *)self);
----------------------------------------------------------------------------------------------

Back home I tried the two forum methods provided
I hope the official website can find out the problem.
